src/main/java/appeng/client/gui/implementations/GuiCraftingStatus.java

```diff
@@ -138,11 +156,21 @@ public void initGui() {
                 0,
                 this.guiLeft + 8,
                 this.guiTop + this.ySize - 25,
-                150,
+                100,
                 20,
                 GuiText.CraftingCPU.getLocal() + ": " + GuiText.NoCraftingCPUs);
         this.buttonList.add(this.selectCPU);
 
+        this.follow = new GuiAeButton(
+                1,
+                this.guiLeft + 111,
+                this.guiTop + this.ySize - 25,
+                50,
+                20,
+                GuiText.Unfollow.getLocal(),
+                ButtonToolTips.Unfollow.getLocal());
```

Comment:
By default button should display "Follow" message, not "Unfollow".

Reply:
Now the bottom is display current following status, You means change it to like when you unfollow it shows follow?

Reply:
like change it from current status to next status

Reply:
@MCTBL This button shouldn't show the *current status*, but the *action* to perform. If I'm *not* following, I want to see the *"Follow"* button. And vice versa.

Reply:
i change them all from `Follow` and `Unfollow` to `ToFollow` and `ToUnfollow`